ui/ui-frontend/projects/referential/src/app/file-format/file-format-preview/file-format-information-tab/file-format-information-tab.component.ts

```diff
@@ -181,5 +204,24 @@ export class FileFormatInformationTabComponent {
 
   resetForm(fileFormat: FileFormat) {
     this.form.reset(fileFormat, { emitEvent: false });
+    const DATE_FORMAT = 'dd/MM/yyyy HH:mm:ss';
+    this.form.controls.createdDate.setValue(formatDate(fileFormat.createdDate, DATE_FORMAT, this.locale));
+    this.form.controls.updateDate.setValue(formatDate(fileFormat.updateDate, DATE_FORMAT, this.locale));
```

Comment:
Ce ne serait pas mieux d'appeler form.reset avec en paramètre le fileFormat modifié pour avoir des dates formatées ?
Ex :
```typescript
this.form.reset({...fileFormat, createDate: formatDate(...), updateDate: formatDate(...)], { emitEvent: false });
```

Reply:
Fait.

Reply:
Finalement non, tu as gardé le code original en ajoutant des ifs ?

Reply:
C'est moi qui ai ajouté les ifs, les valeurs sont parfois undefined et ça plantait.

Reply:
OK, mais on pourrait faire: : `this.form.reset({...fileFormat, createDate: fileFormat.createdDate ? formatDate(...) : undefined, updateDate: fileFormat.updateDate ? formatDate(...) : undefined], { emitEvent: false });`
Bref, peu importe